daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/VariableMap1.scala

```diff
@@ -363,7 +386,7 @@ class VariableMap private(vTable: Map[GlobalQName, ArrayBuffer[VariableInstance]
         // have a defined value
         case VariableDefined if (!variable.value.isDefined && variable.defaultValueExpr.isDefined) => {
           Assert.invariant(maybeState.isDefined)
-          variable.setState(VariableInProcess)
+          variable.setState(VariableBeingDefined)
```

Comment:
This makes me thing that this two states are actually basically the same thing? If not, the distinction isn't very clear.